addons/mass_mailing/wizard/mail_compose_message.py

```diff
@@ -49,6 +49,19 @@ def _prepare_mail_values(self, res_ids):
                 )
                 if body:
                     mail_values['body_html'] = body
+            if mail_values.get('body'):
+                mail_values['body'] = Markup(
+                    '<div><span>{mailing_sent_message}</span></div>'
+                    '<blockquote class="border-start" data-o-mail-quote="1" data-o-mail-quote-node="1">'
```

Comment:
Inside a blockquote ? Isn't it going to have a strange display (chatter is already a bit narrow, if we add margins and stuff from blockquotes) ? Anyway not really important I agree but voila :)